lib/s3routes/routesUtils.js

```diff
@@ -376,10 +376,13 @@ const routesUtils = {
      * @param {array | undefined} range - range in format of [start, end]
      * if range header contained in request or undefined if not
      * @param {object} log - Werelogs logger
+     * @param {object} [extraParams] - object containing extra params
```

Comment:
it's needed to supply any additional information, like the bucket name or formatted range, if needed for the backend to stream data.